response/page_path_test.go

```diff
@@ -0,0 +1,265 @@
+package response
+
+import (
+	"context"
+	"testing"
+
+	. "github.com/smartystreets/goconvey/convey"
+)
+
+type TestCase struct {
+	uri      string
+	expected string
+}
+
+func TestGetPagePath(t *testing.T) {
+	Convey("Given a list of URIs", t, func() {
+		ctx := context.Background()
+
+		visualisationsEndpoints := []TestCase{
```

Comment:
maybe add 2 test for very unlikely paths : empty string -> "", slash only "/", two slashes "//", and null

Reply:
`nil` is impossible, but it would be a good idea to have another list of test cases for the behaviour when none of the previous rules apply: to return the URI unmodified. It would also be good to add some real URLs there.